lib/Telemetry/src/Dispatcher.h

```diff
@@ -0,0 +1,60 @@
+#include "Particle.h"
+#include "DataQueue.h"
+#include "IntervalLogger.h"
+
+#ifndef _DISPATCHER_H_
+#define _DISPATCHER_H_
+
+/**
+ * Dispatcher owns and operates on a collection of loggers
+ **/
+class Dispatcher {
+    public:
+        /**
+         * Constructs a Dispatcher with loggers, numLoggers and DataQueue
+         **/
+        Dispatcher(IntervalLogger **loggers, uint16_t numLoggers, DataQueue *dataQ) {
```

Comment:
Since we plan to only have a single DataQueue and Dispatcher per project, can we just create a DataQueue object in Dispatcher instead of passing it in?

EDIT: Actually forget this, I realize DataQueue is still needed in the main file to actually publish for now.

Reply:
^^ I actually did this but then removed it for that very reason.  I think eventually that should be how it works